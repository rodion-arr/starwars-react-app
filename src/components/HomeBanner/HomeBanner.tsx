import './HomeBanner.scss';
import React from "react";

export const HomeBanner = (): JSX.Element => {
  return (
    <div className="home-banner content-wrapper__item--full-width">
      <div className="home-banner__logo"/>
      <div className="home-banner__text">Welcome to StarWars catalog!</div>
      <div className="home-banner__scroll scroll-animation" title="Scroll down for more content!">
        <div className="scroll-animation__chevron" />
        <div className="scroll-animation__chevron" />
        <div className="scroll-animation__chevron" />
      </div>
    </div>
  );
};
