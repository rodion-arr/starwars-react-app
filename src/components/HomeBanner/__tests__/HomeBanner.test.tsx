import {render} from "@testing-library/react";
import {HomeBanner} from "../HomeBanner";

describe('HomeBanner', () => {
  it('should render correctly', function () {
    const {container} = render(<HomeBanner />);

    expect(container).toMatchSnapshot();
  });
});
