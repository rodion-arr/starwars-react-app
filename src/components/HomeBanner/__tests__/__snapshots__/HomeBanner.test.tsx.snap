// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeBanner should render correctly 1`] = `
<div>
  <div
    class="home-banner content-wrapper__item--full-width"
  >
    <div
      class="home-banner__logo"
    />
    <div
      class="home-banner__text"
    >
      Welcome to StarWars catalog!
    </div>
    <div
      class="home-banner__scroll scroll-animation"
      title="Scroll down for more content!"
    >
      <div
        class="scroll-animation__chevron"
      />
      <div
        class="scroll-animation__chevron"
      />
      <div
        class="scroll-animation__chevron"
      />
    </div>
  </div>
</div>
`;
